internal/testutil/runmain.go

```diff
@@ -102,3 +106,35 @@ func (r *Runner) Run(env map[string]string, f func()) {
 	case <-done:
 	}
 }
+
+// RunNonInteractive runs the build binary until terminated or timeout has
+// been reached, and indicates successful execution on return.
+func (r *Runner) RunNonInteractive(env map[string]string, timeout time.Duration) bool {
```

Comment:
I'm good to rewrite `Run` if it's not used anywhere.

Reply:
also done